src/OrleansAzureUtils/Hosting/AzureSilo.cs

```diff
@@ -174,6 +169,8 @@ public bool Start(ClusterConfiguration config, string deploymentId = null, strin
 
             // Bootstrap this Orleans silo instance
 
+            var deploymentId = host.Config.Globals.DeploymentId;
```

Comment:
>var deploymentId = host.Config.Globals.DeploymentId; [](start = 12, length = 52)

maybe you can keep the method signature as is, and here do:
`deploymentId = deploymentId ?? host.Config.Globals.DeploymentId ?? serviceRuntimeWrapper.DeploymentId;`

This way the change will not be breaking, just doing fallbacks